### ISSUE TITLE
Add dark theme variant option for snippet images.

### ISSUE THREAD

**Issue by glogiotatidis** [mozmeao/snippets-service#930]
tl;dr Some snippet images to do not look great when in the dark theme. Allow the snippet editors to define another set of images for dark themes, in addition to the original images.

Depends on:
 - ~~#841~~ 
 - ~~#942 - Dark theme preview~~
 - ~~#863 - https://bugzilla.mozilla.org/show_bug.cgi?id=1510335~~
 - https://bugzilla.mozilla.org/show_bug.cgi?id=1522166

**Comment by glogiotatidis** [comment]
Work from the ASR side has not landed yet. 

/remind me to check in 2 months

**Comment by glogiotatidis** [comment]
ASR PR is moving. 

/remind me to check in a month

**Comment by glogiotatidis** [comment]
Also we probably want to implement this as an extra `ImageField` in the `Icon` model and automatically populate the `_dark_theme` icon variables.

**Comment by glogiotatidis** [comment]
Per discussion with OMN team this is not a priority atm.

**Comment by glogiotatidis** [comment]
AS bug has been fixed and this is now unblocked.

**Comment by glogiotatidis** [comment]
Discussed with OMN team and this is not a priority. We currently have dark theme preview (#942 and #993) and that's good enough to select a good image that works on both light and dark themes
